heaps/heap_sort.py

```diff
@@ -1,8 +1,18 @@
-
+from heaps.min_heap import MinHeap
 
 def heap_sort(list):
     """ This method uses a heap to sort an array.
-        Time Complexity:  ?
-        Space Complexity: ?
+        Time Complexity:  O(n)
+        Space Complexity: O(1)
     """
```

Comment:
👍  The time complexity is O(n log n) because you're adding n nodes to the heap and each addition takes log n time.  

Similarly the space complexity is O(n) because you build a heap of n nodes.